styles/nixos.wiki/catppuccin.user.css

```diff
@@ -322,6 +322,40 @@
     #footer {
       background-color: @mantle;
     }
+    
+    .mw-search-profile-tabs, {
+      background-color: @base;
+      border: none;
+    }
+    
+    .search-types .current a {
+      color: @text;
+    }
+    
+    .oo-ui-inputWidget-input {
+       background-color: @subtext1 !important;
+       color: @base;
+    }
+    
+    .oo-ui-buttonElement-button {
+       background-color: @blue !important;
+       border: none;
+     }
+    
+    .oo-ui-labelElement-label {
+       color: @base !important;
+    }
+
+    & when (@lookup = latte) {
```

Comment:
try make latte the same as the dark flavours otherwise should be fine...

Reply:
the search bar breaks when i switch to latte so

Reply:
i'll keep it